app/views/field_slips/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<p style="color: green"><%= notice %></p>
+
+<%= render @field_slip %>
+
+<div>
+  <%= link_to "Edit this field slip", edit_field_slip_path(@field_slip) %> |
+  <%= link_to "Back to field slips", field_slips_path %>
+
+  <%= button_to "Destroy this field slip", @field_slip, method: :delete %>
+</div>
```

Comment:
Idk if you'd want the index field slips to also show these "edit/destroy" links per field slip, but it might be possible to pass a "context" local, and put those icon-links, plus a "back to index" link `if context == :show`... Something like that.

Can provide syntax for the icon links if you want to do it that way.

Reply:
I don't think this is needed so I'm not going to sweat it.